src/components/AppliedFilters.tsx

```diff
@@ -14,8 +14,8 @@ import { useStateUpdatedOnSearch } from '../hooks/useStateUpdatedOnSearch';
  * @public
  */
 export interface AppliedFiltersCssClasses {
-  appliedFiltersContainer?: string,
-  appliedFiltersContainer___loading?: string,
+  container?: string,
+  container___loading?: string,
```

Comment:
@tmeyer2115 do you remember the reason we use three underscores here? I think Connor was the one who originally had the idea and but I can't remember exactly what it was